lib/task/import/csvAuthorityRecordRelationImportTask.class.php

```diff
@@ -169,9 +170,13 @@ private function import($filepath, $indexDuringImport = false, $updateMode = fal
 
   private function importRow($sourceActorId, $targetActorId, $relationTypeId)
   {
-    if (!empty($this->import->status['updateMode']))
+    $updateMode = !empty($this->import->status['updateMode'])
+                    ? $this->import->status['updateMode']
+                    : false;
+
+    if (!empty($updateMode))
```

Comment:
This can be be simplified to `if ($updateMode)` now that you are assigning `false` when the flag is empty above.